doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -347,6 +347,13 @@ include::../structs/VkDeviceCreateInfo.txt[]
 
 include::../validity/structs/VkDeviceCreateInfo.txt[]
 
+fname:vkCreateDevice creates a VkDevice object, then enables
+extensions requested by the application. If an extension
+is provided by a layer, the layer must: be specified at
```

Comment:
Hmm..., **must** it really? Or should it instead in that case return the **VK_ERROR_EXTENSION_NOT_PRESENT**.

Now I see the same thing is in the description `vkCreateInstance`.
And it is relevant to the BTW item 2 in issue #259 (is VK_ERROR_EXTENSION_NOT_PRESENT supposed to catch all problems or not?).